gridsome.client.js

```diff
@@ -64,30 +64,46 @@ export default function (Vue, options, { appOptions, router }) {
     next()
   })
 
-  // Maintain path prefix during router change
+  function conditionalLangAttrUpdate(updateFunction) {
+      const lang = i18n.locale || options.defaultLocale;
+      updateFunction(lang);
+  }
+
+// Maintain path prefix during router change
   router.beforeResolve((to, from, next) => {
     // do not rewrite build paths
     if (process.isServer) {
       next()
-      return 
+      return
     }
 
     // if option is disabled skip whole logic
     if (options.enablePathRewrite === false) {
       next()
-      return 
+      return
     }
 
+    // On route load, set the correct lang attribute for html tag using the current locale
+    conditionalLangAttrUpdate((lang) => head.htmlAttrs = {lang: lang});
```

Comment:
Should this maybe spread `head.htmlAttrs` inside the new object in case it is not empty? Not sure about this but some other plugin or core might have added something and with this we would loose everything except for the new `lang`, no?

Reply:
Good point! I'll make these changes soon.